Sources/HummingbirdPostgres/PostgresPersistDriver.swift

```diff
@@ -98,15 +98,27 @@ public final class PostgresPersistDriver: PersistDriver {
 
     /// Set value for key.
     public func set(key: String, value: some Codable, expires: Duration?) async throws {
-        let expires = expires.map { Date.now + Double($0.components.seconds) } ?? Date.distantFuture
-        try await self.client.query(
-            """
-            INSERT INTO _hb_pg_persist (id, data, expires) VALUES (\(key), \(WrapperObject(value)), \(expires))
-            ON CONFLICT (id)
-            DO UPDATE SET data = \(WrapperObject(value)), expires = \(expires)
-            """,
-            logger: self.logger
-        )
+        if let expires {
+            let expires = Date.now + Double(expires.components.seconds)
+            try await self.client.query(
+                """
+                INSERT INTO _hb_pg_persist (id, data, expires) VALUES (\(key), \(WrapperObject(value)), \(expires))
```

Comment:
Besides user session, are there any other use cases for the persist store? The data column is of type json where the content is stored as string instead of `JSONB` which is "more efficient".

Reply:
It is a general purpose key/value store that anyone can use. Sessions is the one thing that Hummingbird uses it for.

My understanding is JSONB is more efficient if you want to index the contents of the json, but here we are just storing it and returning it

Reply:
Understood! Would the use of the [HSTORE](https://www.postgresql.org/docs/current/hstore.html) extension not a good fit for this use case?

Reply:
I guess you could use it, but what advantages does it have over a table? Am I correct in assuming this wouldn't be available in a vanilla install of Postgres.

Reply:
How does the expire date work here with HStore

Reply:
We can talk about this in the future, vanilla install of Postgres still comes with a bunch of extensions (Crypto, vectors to name a few), [they can be enabled with the followings](https://www.postgresql.org/docs/current/sql-createextension.html) `CREATE EXTENSION {extension_name};` Third party extensions such as PGVector are not available by default though. 

As for expire date with HStore, it would be an expired_at could be a property of the attributes.